docs/zephyr_02_functional_requirements.sdoc

```diff
@@ -1174,28 +1174,128 @@ DISCUSSION_DATE: >>>
 [/SECTION]
 
 [SECTION]
-TITLE: Mutex
+TITLE: Thread Syncronization
 
 [REQUIREMENT]
-UID: ZEP-91
+UID: ZEP-MUTEX-1
 STATUS: Draft
 TYPE: Functional
 COMPONENT: Mutex
-TITLE: Mutex Kernel Object
+TITLE: Initialization of Mutex
 STATEMENT: >>>
-Zephyr shall support resource synchronization. (Note synchronization can be for memory access, and mutex may be one implementation, but not the only one).
+the Zephyr RTOS shall provide a mechanism for defining and initializing mutexes.
 <<<
-USER_STORY: >>>
-As a Zephyr OS user I want to be able to sychonize threads when accessing common resources, where the thread shall have the option to:
-- wait eternally until the resources becomes available.
-- immediately return with a negative message if the resource is not available and allow the thread to continue.
-- wait for a given time for the resource to become available or return with a negative message.
+
+[REQUIREMENT]
+UID: ZEP-MUTEX-2
+STATUS: Draft
+TYPE: Functional
+COMPONENT: Mutex
+TITLE: Locking and unlocking of a Mutex
```

Comment:
Would it make sense to split locking and unlocking in two different requirements?

Reply:
done